mlrun/runtimes/nuclio/function.py

```diff
@@ -698,7 +698,7 @@ def disable_default_http_trigger(
         """
         self.spec.disable_default_http_trigger = True
 
-    @min_nuclio_versions("1.12.8")
+    @min_nuclio_versions("1.13.1")
```

Comment:
should be able to do `@min_nuclio_versions("1.12.8", "1.13.1")`

Reply:
it will work but what's the benefit?

Reply:
if users have nuclio 1.12.9 shouldn't this be supported? if you remove `1.12.8` it will require users to upgrade to 1.13.1

Reply:
second point is a bug.. look at the previous implementation https://github.com/mlrun/mlrun/commit/87244882c16090977448d7b1500842d3732921fe#diff-91002e128e55ae0b16d8d62abbd2c9f54e83d5d584a287b43f0df436df7b2f88L74